tests/Tests.AzureAppConfiguration/PushRefreshTests.cs

```diff
@@ -0,0 +1,334 @@
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken2"
+                                    }
+            };
+
+        List<PushNotification> invalidPushNotificationList = new List<PushNotification>
+            {
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1;sn=001"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+									EventType = null,
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store1.resource.io/kv/searchQuery1"),
+									EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = null,
+                                    SyncToken = null
+                                    }
+            };
+
+		Dictionary<string, EventGridEvent> eventGridEventList = new Dictionary<string, EventGridEvent>
```

Comment:
>eventGridEventList [](http://example.com/codeflow?start=37&length=18)

Its not a list. Also, we should avoid adding data type to the variable names. You can name it somethign like `eventGridEvents`